### PR TITLE
#39113 Replace Datetime with VirtualDateTime

### PR DESCRIPTION
**Pull request by raamSoftwire**

### CODE REVIEW

File: WarmUp/Program.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using GenderPayGap.Extensions;
```
Comment (jamesgriff) [inline WarmUp/Program.cs]: The WarmUp script is its own, entirely separate application.
We run it on the build server, not in a context that has the configuration necessary to change the VirtualDateTime.
So, for WarmUp, I think we shouldn't use VirtualDateTime.

File: WarmUp/WarmUp.csproj [hunk 2]
```diff
@@ -5,4 +5,8 @@
     <TargetFramework>netcoreapp2.2</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\GenderPayGap.Extensions\GenderPayGap.Extensions.csproj" />
+  </ItemGroup>
+
```
Comment (jamesgriff) [inline WarmUp/WarmUp.csproj]: Similarly to my other comment, I think WarmUp shouldn't depend on the other projects.

File: WarmUp/Program.cs (hunk 1 above)
Comment (raamSoftwire) [reply WarmUp/Program.cs]: Good point, I've reverted these changes and force pushed

File: WarmUp/WarmUp.csproj (hunk 2 above)
Comment (raamSoftwire) [reply WarmUp/WarmUp.csproj]: Fixed

**Comment by jamesgriff** [comment]
@raamSoftwire Once you're done testing the Reminder Email ticket, remember to change that class to use `VirtualDateTime`

**Comment by raamSoftwire** [comment]
Will do, I've made a note to sort this

**Pull request merged**